R/safety.R

```diff
@@ -1,4 +1,11 @@
 first_time <- function(path) {
+  description <- file.path(path, "DESCRIPTION")
+  if (!file.exists(description)) {
+    stop("Could not find a DESCRIPTION file in ", path, ". ",
```

Comment:
Per http://style.tidyverse.org/error-messages.html#problem-statement, can you please reword to use "must"?

Reply:
Yup! Updated the message.